webapp/channels/src/components/file_preview_modal/file_preview_modal.tsx

```diff
@@ -326,7 +325,7 @@ export default class FilePreviewModal extends React.PureComponent<Props, State>
                         >
                             <React.Suspense fallback={null}>
                                 <PDFPreview
-                                    fileInfo={fileInfo}
+                                    fileInfo={fileInfo as FileInfo}
```

Comment:
Why do we need this type assertion here? I would expect to not need this kind of assertions.

Reply:
Yess Even i thought so! i dono why typescript it not recognising that!
And also in that file everywhere this type assertion is made when used `fileInfo`
![image](https://github.com/mattermost/mattermost/assets/93595710/bf4b95f5-f1ca-40bd-a4ab-81400081cb35)